app/views/articles/edit.html.erb

```diff
@@ -28,5 +28,6 @@
 <% end %>
 
 <%= render "shared/webcomponents_loader_script" %>
-<%= javascript_packs_with_chunks_tag "clipboardCopy", "articleForm", defer: true %>
 <%= render "articles/v2_form", article: @article, organizations: @organizations, version: @version %>
+
+<%= javascript_packs_with_chunks_tag "clipboardCopy", "articleForm", defer: true %>
```

Comment:
Changing the order shouldn't matter since it is deferred. Can you consistently get it working with this change in order?

Reply:
I couldn't reproduce the error even before. It only happens to some people and not constantly :-/

Reply:
The "nothing appears" bug was hard to be reproduce with or without the fix :(
The "short textarea" bug is reproduced pretty consistently for me with or without the fix :(